build/noderesourcetopology-plugin/Dockerfile.tools

```diff
@@ -26,9 +26,9 @@ RUN mkdir -p $HOME && \
     tar -C /usr/local -xzf go.tar.gz && \
     rm go.tar.gz && \
     # get required golang tools and OC client
-    go get github.com/onsi/ginkgo/ginkgo && \
-    go get golang.org/x/lint/golint && \
-    go get github.com/mattn/goveralls && \
+    go install github.com/onsi/ginkgo/ginkgo && \
+    go install golang.org/x/lint/golint && \
```

Comment:
We need one more/last change to unblock this PR and the final transition to 1.18.
until we figure out the version to pin, let's use
```
go install golang.org/x/lint/golint@latest
```
(here and below `@latest`) 
xref: https://github.com/openshift-kni/numaresources-operator/blob/main/Dockerfile.tools#L29

Reply:
Thanks, updating now.